cypress/lib/helper.js

```diff
@@ -35,4 +35,19 @@ export function checkProspectsAreVisible() {
   cy.contains(prospect1)
   cy.contains("Prospect 2")
   cy.contains("Prospect 10")
+}
+
+export function checkBookingsOrder(bookingNumber, ordering) {
+  /* get all bookings cards and check their text one by one.
+    We have to check the text using the multiple of 5 because 
+    each card has 5 lines */
+  let lineNumber = 0
+  const cardLinesNumber = 5
+  cy.get('p').each(function($booking) {
```

Comment:
is there a way to do a map on the data from the booking and check if the arrays are equal?

Reply:
I do not understand what you mean. which arrays should be equal and with whom?

Reply:
It doesn't matter because we'll refactor the function